DESCRIPTION

```diff
@@ -16,12 +16,11 @@ Encoding: UTF-8
 Language: en-US
 LazyData: true
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.2.1
+RoxygenNote: 7.2.3
 Depends: R (>= 3.5)
 Imports:
     admiral,
     admiraldev,
```

Comment:
So we have some compatibility issues - we should have admiral (>= 0.10.0) and admiraldev (>=0.3).